brian2tools/baseexport/collector.py

```diff
@@ -226,6 +260,17 @@ def collect_PoissonGroup(poisson_grp):
 
     # get rates (can be Quantity or str)
     poisson_grp_dict['rates'] = poisson_grp._rates
+    if type(poisson_grp._rates) == str:
```

Comment:
Idiomatically this should rather be `if isinstance(poisson_grp._rates, str)`, even though it won't make any difference in practice.